backend/reservations/views/admin.py

```diff
@@ -57,15 +57,15 @@ def assign_room(type_purchased_secpty):
 
 def guest_contact_new(guest_new, otp, email_onboarding=False, room=None):
     ''' Create guest send email '''
-    logger.info(f"[+] Creating guest: {guest_new['first_name']} {guest_new['last_name']}, {guest_new['email']}, {guest_new['ticket_code']}")
+    logger.info("Creating guest: %s %s (%s) %s", guest_new['first_name'], guest_new['last_name'], guest_new['email'], guest_new['ticket_code'])
```

Comment:
Man, I keep twitching when I see these rather than f-strings.

Reply:
Because both you and @Index01 mentioned it I poked into this a bit. When it comes to logging it's an [optimization thing](https://docs.python.org/3/howto/logging.html#optimization) but I'm gonna be switching to f-strings on basically anything else and any other project. Long time coming given PEP498 came out in '15 heh.